Algorithm/QCAlgorithm.Python.cs

```diff
@@ -141,11 +141,12 @@ public Security AddData(PyObject type, Symbol underlying, Resolution resolution,
         /// <returns>The new <see cref="Security"/></returns>
         public Security AddData(Type dataType, string ticker, Resolution resolution, DateTimeZone timeZone, bool fillDataForward = false, decimal leverage = 1.0m)
         {
+            // NOTE: Invoking methods on BaseData w/out setting the symbol may provide unexpected behavior
```

Comment:
I didn't want to change behavior here, but left a note since some implementations of `RequiresMapping()` require the symbol object to be set.